### ISSUE TITLE
Message in bold or italic format

### ISSUE THREAD

**Issue by demo-design** [microsoft/banner-settings-ado-extension#11]
Can we write the message( info,warning,error) in bold or italic format?? 
and can we add multiple banners in different projects??

**Comment by Glenn-Gidlof** [comment]
Yes, adding the ability to use Italic and / or bold would be very helpful
